sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -157,44 +158,40 @@ def read(
             request_options["populatePartitionKeyRangeStatistics"] = populate_partition_key_range_statistics
         if populate_quota_info is not None:
             request_options["populateQuotaInfo"] = populate_quota_info
-
         collection_link = self.container_link
         self._properties = self.client_connection.ReadContainer(
             collection_link, options=request_options, **kwargs
         )
-
         if response_hook:
             response_hook(self.client_connection.last_response_headers, self._properties)
-
         return cast('Dict[str, Any]', self._properties)
 
     @distributed_trace
     def read_item(
         self,
         item,  # type: Union[str, Dict[str, Any]]
         partition_key,  # type: Any
-        populate_query_metrics=None,  # type: Optional[bool]
+        populate_query_metrics=None,  # type: Optional[bool] # pylint:disable=docstring-missing-param
         post_trigger_include=None,  # type: Optional[str]
         **kwargs  # type: Any
     ):
         # type: (...) -> Dict[str, Any]
         """Get the item identified by `item`.
-
         :param item: The ID (name) or dict representing item to retrieve.
+        :type item: Union[str, Dict[str, Any]]
```

Comment:
I think sphinx actually prefers the `or` syntax over the `Union[]` syntax. Would be worth quickly checking the rendered docs to see if they look okay using `Union[]` in the docstrings.

Reply:
no problems as far as I've seen:
<img width="517" alt="image" src="https://github.com/Azure/azure-sdk-for-python/assets/30335873/e757ef2e-11f4-42a1-8d1e-e5988c43dfe0">